cmd/server/config.go

```diff
@@ -11,9 +10,12 @@ import (
 )
 
 const (
-	ocidRoot            = "/var/lib/ocid"
+	ocidRoot            = "/var/lib/containers"
+	ocidRunRoot         = "/var/run/containers"
 	conmonPath          = "/usr/libexec/ocid/conmon"
-	pausePath           = "/usr/libexec/ocid/pause"
+	pauseImage          = "kubernetes/pause"
```

Comment:
:+1: We can revisit this later (after this PR hits). In particular, a proper full-on solution would involve being able to alias images to a local directory which has OCI layouts in it -- so you could still specify `docker://abc` and that would map to some image already on the filesystem.